docs/graphql/manual/guides/deployment/digital-ocean-one-click.rst

```diff
@@ -301,6 +301,16 @@ database.
   to configure all required permissions (not applicable with the default
   connection string with DO Managed Postgres).
 
+Connection Pooling
+------------------
+
+  Connection Pooling is built-in feature of graphql-engine. The default connection pool size is 50.
+  If you need to configure pool size or timout you can use below environment variables.
```

Comment:
If you need to configure **the** pool size or **the** timeout, you can use **the** below environment variables.

Reply:
Done.